Add optional Vampir auto-launch integration

OutlyingWest opened the pull request:
Adds optional Vampir launching functionality. When enabled by a magic command, the kernel will automatically launch Vampir on the generated traces.otf2 file after Score-P instrumented cell execution.

### Features
`%%enable_vampir_launch_on_scorep_instrumented` magic:

Enables Vampir auto-launch after the next Score-P instrumented execution.

`%%disable_vampir_launch` magic:

Disables Vampir auto-launch.

Commit:
vampir launch command added

Commit:
vampir disabling command added

Commit:
vampir launch control commands documented; logging chapter moved to Usage

Commit:
tests fixed

elwer commented:
Works great!
Can you check in the `%%enable_vampir_launch_on_scorep_instrumented` already whether vampir is on $PATH? If it is not, provide a hint that it has to be added in order to enable the loading feature, e.g. via adding it with ` %env` magic command.

Besides that, please check for consistency in line and cell magics, as discussed.

Commit:
ignore scorep-* results

Commit:
VAMPIR_NOT_FOUND message at %%enable_vampir_launch_on_scorep_instrumented execution

Commit:
%env unable to unwrap vars like '$PATH' -> not found PATH message fixed

OutlyingWest commented:
> Works great! Can you check in the `%%enable_vampir_launch_on_scorep_instrumented` already whether vampir is on $PATH? If it is not, provide a hint that it has to be added in order to enable the loading feature, e.g. via adding it with ` %env` magic command.
> 
> Besides that, please check for consistency in line and cell magics, as discussed.

I added check in the %%enable_vampir_launch_on_scorep_instrumented fcc92dc and changed "not found in PATH" message in 29a7570

Commit:
os unused deleted
